chain/jsonrpc/src/lib.rs

```diff
@@ -632,24 +637,17 @@ impl JsonRpcHandler {
         &self,
         request_data: near_jsonrpc_primitives::types::blocks::RpcBlockRequest,
     ) -> Result<near_jsonrpc_primitives::types::blocks::RpcBlockResponse, RpcBlockError> {
```

Comment:
Following the style, we should use absolute path:

```suggestion
    ) -> Result<near_jsonrpc_primitives::types::blocks::RpcBlockResponse, near_jsonrpc_primitives::types::blocks::RpcBlockError> {
```

P.S. We may consider `use near_jsonrpc_primitives::types as near_jsonrpc_types;` at the top to shorten the lines a bit, if that helps at all.